data/campaigns/Winds_of_Fate/scenarios/01_The_Hunt.cfg

```diff
@@ -16,7 +16,8 @@
 
     [story]
         [part]
-            #po: In this campaign, the narrator and drake characters speak with the Morogor dialect, which has some unique syntax and terms. Please read story/Morogor_Dialect before translating and refer back to it as needed.
+            #po: In this campaign, the (sometimes biased and unreliable) drake narrator and drake characters speak with the Morogor dialect, which has some unique syntax and terms.
```

Comment:
It is enough to say _"the (sometimes biased) drake narrator"_ and leave out the redundant "unreliable". Other than that it looks good to merge.

Reply:
Is the "unreliable" doing any harm? If I read a translation hint that said just "biased narrator", then I'd turn to the internet to find out what a "biased narrator" is, on the assumption that, if the writer had meant an "unreliable narrator" then they would have used the common term.

Reply:
If we want to invoke the (Anglosphere) term-of-art `"Unreliable Narrator"` then it needs to be those two words together, probably capitalized and possibly linking to the wikipedia article on the subject (or another online encyclopedic reference). By itself, the word "unreliable" does not necessarily convey that specific meaning.

However, having reread through the whole narration with this PR's changes, the wording "the (sometimes biased) drake narrator" seems to be a more specific and clear description of the narration technique. Whereas "Unreliable Narrator" can have broader implications, such as a narrator character outright lying about events or directly contradicting what happens in the scenarios. Since nothing that crude happens with the narration, it might mislead translators into hunting for a bunch of discrepancies which are not there.

Reply:
While it's clear to you what "the (sometimes biased) drake narrator" means, if "unreliable" isn't a synonym then it's not clear to me. Please assume it won't be clear to all of the translators either, and expand on the description of what you want to tell them.

Reply:
> Please assume it won't be clear to all of the translators either, and expand on the description of what you want to tell them.

Alright, how about just a simpler approach then:

`#po: In this campaign, the drake narrator and other drake characters speak with the Morogor dialect, which has some unique syntax and terms.`

So translators know the narrator is a drake character, which is all the information relevant to the current narration. And all that I was trying to make clear to them when I raised the issue in the first place.

Reply:
The latest push uses @Jonathan-Kelly's simpler approach.

Reply:
> The latest push uses @Jonathan-Kelly's simpler approach.

Wow, it moved from highlighting "racist" views to "biased" views to finally say nothing cause it is "simpler".
If this is not mainstreaming racist views, I don't know what it is. Good job guys!

Reply:
What have I missed @demario-w? At the moment, I think it just says that they've a dialect with an unusual grammar and terms such as "Contention", in a context where the intro explains that they've been isolated from other speakers of the common tongue for a long time.

Reply:
> What have I missed @demario-w?

Mmm. I think you've have seen everything happening in front of you:
- you raise a concern about racist views
- the author [convinced](https://github.com/wesnoth/wesnoth/pull/8124#issuecomment-1873439883) you that it is not racist but barely "opinionated" (hint: racism is not an opinion)
- as you are giving in, he then [convinced](https://github.com/wesnoth/wesnoth/pull/8124#discussion_r1452835711) you that is was not "opinionated" but barely "biased" (hint: bias is unconscious)
- as you agree on "biased", he [convinced](https://github.com/wesnoth/wesnoth/pull/8124#discussion_r1453955712) you that adding nothing was best. Again you gave in.

What is left of your original concern?

> At the moment, I think it just says that they've a dialect with an unusual grammar and terms such as "Contention", 

The "drake dialect" is just the same thing as the "pure fiction", allowing the author to push racist views in wesnoth mainline.

Come on, "noble creatures"? When are we reaching the point where "[master race](https://en.wikipedia.org/wiki/Master_race)" and "[untermenschen](https://en.wikipedia.org/wiki/Untermensch)" is all good fun?

Reply:
Don't forget that the actual racist reference was removed. What's being discussed here is just a translator hint about the nature of the narrator. I don't think the "simpler approach" makes any sense either, but it's tangential to the original racist concern.

It's definitely true that "noble creatures" could be reminiscent of a "master race" concept. It doesn't feel like it's heading that way in this case, since it refers to unknown creatures… though the fact that it's being distinguished with "legions" is perhaps slightly suspicious.

Reply:
The actual text shown to the player has improved. Not to the extent that I'd like it to, but sufficiently that I see Karron's actions as being about gaining territory, rather than a racist motivation.

For the po hint in this thread, I accepted "biased" when CelticMinstrel voted for it, but added "unreliable" because "biased narrator" seems to be a synonym for "unreliable narrator". The version that's "simpler" appears to me to remove all of the stuff about "opinionated" and "biased"; while it doesn't clean up the references to orcs in the Morogor Dialect file, I don't see a problem with a po hint pointing to that file. As for the drake dialect use of "prey", well I think Morogor would have turned to cannibalism if TRoW hadn't happened when it did, so I feel that's okay at the start of the story, and it makes Gorlack's ending be a political progression from the starting point.

For "noble creatures", [nemaara's interpretation of the phrase](https://github.com/wesnoth/wesnoth/pull/8124#discussion_r1449285252) was a complete change from the interpretation that both I and Jonathan-Kelly were arguing over. So there's a po hint to ensure that nemaara's interpretation is used in translations. Part of my reasoning here is that I don't see another race cooperating with the drakes in the way that Jonathan-Kelly's epilogue would have them do, I think all politicians are going to be making contingency plans in case an ally becomes an enemy.

Reply:
> Don't forget that the actual racist reference was removed.

What has been done is to turn explicit racist views into dog-whistlingly racist views.

> It's definitely true that "noble creatures" could be reminiscent of a "master race" concept. It doesn't feel like it's heading that way in this case, since it refers to unknown creatures… 

Throughout the whole campaign, any "race" that the drakes met have been turned into a existential enemy that needed to be "hunt". At which step, did drakes ponder if any of them was "noble" enough to be spared? How does it make sense that they expect to find a new continent with unknown "noble" species that can be turned into allies? It will be just more of the same. **It is in their worldview.**

Reply:
> Throughout the whole campaign, any "race" that the drakes met have been turned into an existential enemy that needed to be "hunt". At which step, did drakes ponder if any of them was "noble" enough to be spared? How does it make sense that they expect to find a new continent with unknown "noble" species that can be turned into allies? It will be just more of the same. **It is in their worldview.**

You have a fair point here. I haven't played the campaign myself, so I can't say if the drakes ever pondered whether any of their opponents were "noble" enough to be spared, though from what I've heard so far about it, it feels like they must've pondered that somewhere along the way.

Reply:
> Throughout the whole campaign, any "race" that the drakes met have been turned into an existential enemy that needed to be "hunt".

While that's a reading that @Jonathan-Kelly has been presenting in this PR, I think that's mostly an addition that, while it fits with the text in the campaign, isn't in the campaign itself. A lot is left in the player's interpretation, and while I'd hope the text will move in the directions that we'd like during 1.19, I think merging this PR is good for 1.18.

Up to and including S06, the main motivation is a lack of food.

For S05, S06 and S07, it's also driven by the Saurian's against the humans and elves; given the events of TRoW this feels more like self-defense, for the Saurians these are existential enemies because of the war that the elves are pursuing against them. Filling in from LoW, among the elves themselves there's a difference of opinion over this that's a major factor in starting the elvish civil war.

S08 is the drakes running away because the elves are chasing them, S09 is a wilderness, S10 is the dwarves who live under that wilderness coming to complain about drake's sewage outflow.

My preferred reading of the epilogue is that Gorlack let Karron take the most warmongering drakes out of the area where most canon stories are set, leaving a calmer set of drakes for the later stories set on the Great Continent.

Reply:
> A lot is left in the player's interpretation, and while I'd hope the text will move in the directions that we'd like during 1.19

The least this project could do before this campaign is release in BfW 1.18 is:

1. if it is agreed that this campaign contains hate speech and racist views, a disclaimer is shown to the player before the campaign is started
2. if the text from this campaign will change from the current form during the 1.19 development cycle, the information is passed to translators so they can make an informed decision to spend time on translation that will be invalided within a year.

Reply:
> > Throughout the whole campaign, any "race" that the drakes met have been turned into an existential enemy that needed to be "hunt".

> While that's a reading that @Jonathan-Kelly has been presenting in this PR, I think that's mostly an addition that, while it fits with the text in the campaign, isn't in the campaign itself.

That is almost the exact opposite of what I have (unsuccessfully) attempted to communicate.

One might go to war with an "existential enemy" but no one would ever try to hunt something that dangerous as a casual food source. Out of desperation, ignorance and supreme arrogance the drakes from morogor failed to make that kind of rational distinction, leading them to catastrophe. How does the story I wrote or anything I said in this PR contradict that?

> Up to and including S06, the main motivation is a lack of food.
> 
> For S05, S06 and S07, it's also driven by the Saurian's against the humans and elves; given the events of TRoW this feels more like self-defense, for the Saurians these are existential enemies because of the war that the elves are pursuing against them.

Correct.

> Filling in from LoW, among the elves themselves there's a difference of opinion over this that's a major factor in starting the elvish civil war.

This seems to be stretching an interpretation of LoW's story a bit thin. While LoW does not say anything that contradicts your interpretation, it does not support it very well either. The Ka'lian council is against [all of the protagonists' actions beyond orders](https://github.com/wesnoth/wesnoth/blob/8da8b96b07bd69fa01e8129b2a6b5307810f2d1d/data/campaigns/Legend_of_Wesmere/scenarios/chapter5/20_Council_Ruling.cfg#L23-L37), not just destroying Saurgrath but saving Tath from destruction and putting the orcs into a state of disarray to end the war.

Landar going insane and eventually assassinating not just the Ka'lian, but off-screen deciding to preemptively kill his old friend Kalenz is what sparks the elvish civil war.

> S10 is the dwarves who live under that wilderness coming to complain about drake's sewage outflow.

Well... I guess, that is a pretty funny interpretation at least. The gist of S10 is the dwarves attack the drakes out of a mutual miscommunication and minor territorial dispute, and after failing to drive the drakes off the mountain, are offered a truce by Gorlack (which they accept). This mirrors Resha's pax with the naga in S4, showing a big development away from "every other species is just a prey item".

> My preferred reading of the epilogue is that Gorlack let Karron take the most warmongering drakes out of the area where most canon stories are set, leaving a calmer set of drakes for the later stories set on the Great Continent.

The more headstrong and/or desperate drakes mostly died or fled ("scattered") into the more remote parts of the Great Continent. This is in case other writers want to have drakes appear as a minor enemy for game play variety, in place of the usual orcs or undead. For 1.18, this supports the newly rewritten EI where there are several encounters with a flight of drakes which still generally refers to other species as "prey".